packages/component/src/Utils/TypeFocusSink/navigableEvent.js

```diff
@@ -0,0 +1,67 @@
+// Navigable event means the end-user is focusing on an inputtable element, but it is okay to capture the arrow keys.
+// For example, if the end-user is on <button>, we can capture the arrow keys.
+// We can also capture arrow keys when the user is on a <textarea> without contents.
+export default function navigableEvent(event) {
+  const {
+    altKey,
+    target,
+    target: { tagName }
+  } = event;
+
+  const autocompleteAttribute = target.getAttribute('autocomplete');
+  const autocomplete = autocompleteAttribute && autocompleteAttribute !== 'off';
+
+  // Generally, we allow up/down arrow keys on all elements captured here, except those handled by the user agent.
```

Comment:
This should also be documented in `ACCESSIBILITY.md`

Reply:
Let me do this in another PR, so you can comment on it more easily.